docs/content/users/install/ddev-installation.md

```diff
@@ -265,17 +264,11 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
     ### WSL2/Docker Desktop Manual Installation
 
     You can manually step through the process the install script attempts to automate:
-
-    1. Install [Chocolatey](https://chocolatey.org/install):
-        ```powershell
-        Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
-        iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))`
-        ```
-    2. In an administrative PowerShell, run `choco install -y ddev mkcert`.
+    1. Download and run the [Windows-side installer](https://github.com/ddev/ddev/releases) (used for hosts-file management only).
     3. In an administrative PowerShell, run `mkcert -install` and follow the prompt to install the Certificate Authority.
     4. In an administrative PowerShell, run `$env:CAROOT="$(mkcert -CAROOT)"; setx CAROOT $env:CAROOT; If ($Env:WSLENV -notlike "*CAROOT/up:*") { $env:WSLENV="CAROOT/up:$env:WSLENV"; setx WSLENV $Env:WSLENV }`. This will set WSL2 to use the Certificate Authority installed on the Windows side. In some cases it takes a reboot to work correctly.
```

Comment:
2. is removed, the list numbers should be updated.

And `mkcert -install`, `mkcert -CAROOT` here in manual instructions, I think it should be `"C:\Program Files\DDEV\mkcert.exe" ...`